vue/components/index.js

```diff
@@ -48,6 +50,7 @@ export {
     SearchPlatforme,
     SidePlatforme,
     TablePlatforme,
+    GenericButtonPlatforme,

```

Comment:
`ButtonPlatforme` is enough since it's the "base case".

Reply:
Já tem um ButtonPlatforme, não haverá conflito com a duplicação?